server/middleware/setUpCurrentUser.ts

```diff
@@ -1,13 +1,10 @@
 import { Router } from 'express'
-import auth from '../authentication/auth'
-import tokenVerifier from '../data/tokenVerification'
 import populateCurrentUser from './populateCurrentUser'
 import type { Services } from '../services'
 import populateCurrentUserRoles from './populateCurrentUserRoles'
 
 export default function setUpCurrentUser({ userService }: Services): Router {
   const router = Router({ mergeParams: true })
-  router.use(auth.authenticationMiddleware(tokenVerifier))
```

Comment:
I think you need this back in otherwise auth tokens won't be verified?

Reply:
Apologies had not checked that file in.